canvas_cli/apps/plugin/plugin.py

```diff
@@ -133,6 +138,29 @@ def init(
     print.json(f"Project created in {project_dir}", project_dir=project_dir)
 
 
+@app.command(short_help="Validates the Canvas Manifest json file of a plugin package")
+def validate(
+    package: Path = typer.Argument(
+        ..., help="Path to a dir containing the python package to install"
+    ),
+) -> None:
+    """Validates the Canvas Manifest json file of a plugin directory."""
+    if not package.exists():
+        raise typer.BadParameter(f"Package {package} does not exist")
+
+    if not package.is_dir():
+        raise typer.BadParameter(f"Package {package} is not a directory, nothing to validate")
+
+    manifest = Path(f"{package.name}/CANVAS_MANIFEST.json")
+    if not manifest.exists():
+        raise typer.BadParameter(
+            f"Package {package} does not have a CANVAS_MANIFEST.json file to validate"
+        )
+
+    validate_manifest_file(json.loads(manifest.read_text()))
+    print.json(f"Package {package} has a valid CANVAS_MANIFEST.json file")
```

Comment:
Is this `print.json` a typo? Should it just be `print`?

Reply:
`print.json` is used in all the other methods so i was just following suit to be consistent!